build.gradle

```diff
@@ -10,7 +10,7 @@ plugins {
 ext {
     wordPressUtilsVersion = '2.7.0'
     wordPressLoginVersion = 'trunk-657bed28fa735780e3ffcc214503a9e5d1286d6c'
-    gutenbergMobileVersion = 'v1.81.0'
+    gutenbergMobileVersion = 'trunk-c39088daab5b49b48db81b01bc1e143999d8396b'
```

Comment:
Friendly reminder: we'll need to change this to a `gutenberg-mobile` alpha release prior to merging this PR.